models/connections.go

```diff
@@ -9,6 +10,50 @@ import (
 	"github.com/google/uuid"
 )
 
+// List of all connection types
+const (
```

Comment:
I thunk these are lowercase

Reply:
These are all saved as uppercase (& some with spaces). It's coming from https://github.com/flanksource/flanksource-ui/blob/main/src/components/Connections/connectionTypes.tsx


```
 SELECT DISTINCT type FROM connections;
+--------------+
| type         |
|--------------|
| Gotify       |
| Zulip Chat   |
| Slack        |
| Google Chat  |
| postgres     |
| Google Cloud |
| AWS          |
| Azure        |
| Discord      |
| Matrix       |
| Email        |
| HTTP         |
| Ntfy         |
| Azure Devops |
| Postgres     |
| Telegram     |
+--------------+
```

Reply:
Although, I do think that these needs to be changed so we only have lowercase and no-spaces.

Reply:
Logged https://github.com/flanksource/flanksource-ui/issues/1448 for that